lib/phase/structural.js

```diff
@@ -68,10 +60,10 @@ module.exports = doc => (
                 return reject(schemaErr);
             }
 
-            const error = checkAdditionalRules(data);
+            const errors = checkAdditionalRules(data);
 
-            if (error) {
-                return reject(error);
+            if (errors) {
+                return reject(errors);
```

Comment:
[ask] What kind of error message shown for user, especially when multiple return object is rejected here?

Reply:
`Error: Job main has duplicate step: preinstall,Job main has duplicate step: pretest,......`

Reply:
👍